C/plugins/storage/postgres/init.sql

```diff
@@ -838,6 +835,7 @@ INSERT INTO foglamp.statistics ( key, description, value, previous_value )
             ( 'BUFFERED',   'The number of readings currently in the FogLAMP buffer', 0, 0 ),
             ( 'SENT_1',     'The number of readings sent to the historian', 0, 0 ),
             ( 'SENT_2',     'The number of statistics data sent to the historian', 0, 0 ),
+            ( 'SENT_3',     'The number of statistics data sent to the http', 0, 0 ),
```

Comment:
Strictly the name is wrong as it is readings and not statistics.

Reply:
fixed